Linux/sgx/test_app/enclave/tests/evp_smx.c

```diff
@@ -36,591 +36,666 @@
 #include "openssl/bio.h"
 #include "openssl/evp.h"
 #include "openssl/ec.h"
-#include "openssl/pem.h" /* openssl library */
+#include "openssl/pem.h"
+#include "openssl/decoder.h"
```

Comment:
It seems `openssl/decoder.h` not a file of OpenSSL 1.1.1, there will be build error

```
enclave/tests/evp_smx.c:40:10: fatal error: openssl/decoder.h: No such file or directory
 #include "openssl/decoder.h"
          ^~~~~~~~~~~~~~~~~~~
compilation terminated.
```